services/cinema_hall.py

```diff
@@ -0,0 +1,19 @@
+from django.db.models import QuerySet
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(
+        hall_name: str = None,
+        hall_rows: int = None,
+        hall_seats_in_row: int = None,
+) -> None:

```

Comment:
The parameters `hall_name`, `hall_rows`, and `hall_seats_in_row` have default values of `None`. This could lead to issues if the function is called without arguments. Consider adding validation to ensure these parameters are provided and are valid (e.g., non-empty string for `hall_name`, positive integers for `hall_rows` and `hall_seats_in_row`).